packages/dashboard-v2/.env.development

```diff
@@ -0,0 +1,2 @@
+GATSBY_PORTAL_DOMAIN=
+GATSBY_HOST=
```

Comment:
https://www.gatsbyjs.com/docs/how-to/local-development/environment-variables/#add-env-files-to-gitignore

Reply:
> Blank .env.development file is added to the repo to ease the onboarding process for anyone who will work on the dashboard locally, but I also marked it to not be tracked anymore.

Reply:
b731d80e